build.sbt

```diff
@@ -110,7 +111,7 @@ lazy val core = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     testSettings,
     moduleName := "metaconfig-core",
     libraryDependencies ++= List(
-      "com.lihaoyi" %%% "pprint" % "0.5.3",
+      "com.lihaoyi" %%% "pprint" % "0.5.5",
```

Comment:
Version 0.5.5 do not support Scala 2.11
https://mvnrepository.com/artifact/com.lihaoyi/pprint

@olafurpg Can we drop the support for Scala 2.11 (for now)?

Reply:
I think the correct solution would be to select and older version of pprint for 2.11

Reply:
But for the fastparse. The api changed a bit and you need to maintain two versions of this library code.

Reply:
@FabioPinheiro yeah, I agree with you on this. I think the most reasonable solution is to drop the `metaconfig-hocon` module (the only one that uses fastparse) entirely and rely on sconfig for cross-platform compatibility.

This would allow us to move forward by dropping an experimental module with known bugs and without dropping 2.11.